kernel/proc.c

```diff
@@ -434,6 +502,37 @@ wait(uint64 addr)
   }
 }
 
+
+
+uint 
+random(void) 
```

Comment:
이 파일에서만 쓰이는거같아서 아래와 같이 하면 좋을거같아요
```suggestion
static uint 
random(void) 
```